packages/backend-common/src/service/lib/config.ts

```diff
@@ -112,7 +120,7 @@ export function readCorsOptions(config: Config): CorsOptions | undefined {
   }
 
   return removeUnknown({
-    origin: getOptionalStringOrStrings(cc, 'origin'),
+    origin: getOptionalGlobOrGlobs(cc, 'origin'),
```

Comment:
The return value here is tied to the express CORS middleware, so it's not quite as generic as the name implies. Perhaps we could do something like `createCorsMatcher(getOptionalStringOrStrings(cc, 'origin'))`?

Reply:
yes, make sense, I wlil update that